qbittorrent_exporter/exporter.py

```diff
@@ -75,11 +68,8 @@ def get_qbittorrent_status_metrics(self):
         try:
             response = self.client.transfer.info
             version = self.client.app.version
-            self.torrents = self.client.torrents.info()
-        except APIConnectionError as e:
-            logger.error(f"Couldn't get server info: {e.error_message}")
```

Comment:
This threw the following error: `AttributeError: 'APIConnectionError' object has no attribute 'error_message'`